man/chpldoc.txt

```diff
@@ -30,8 +28,14 @@ DESCRIPTION
                     sequence used to distinguish a documentation comment
                     from a normal one (defaults to '/*' if unspecified).
 
+  --save-sphinx <directory>  Save generated Sphinx project in directory.
+
   --text-only       Specify that documents should use a text output file
-                    format rather than HTML.
```

Comment:
I might simplify this to something like:

```
Specify that documents should use a text output file format instead of HTML. This option takes precedence over --[no-]html.
```